scripts/mvftoms.py

```diff
@@ -36,6 +37,33 @@
 from katdal.sensordata import pickle_loads
 
 
+def load(dataset, indices, vis, weights, flags):
+    """Load data from one a lazy indexer into existing storage.
+
+    This is optimised for the MVF v4 case where we can use dask directly
+    to eliminate one copy, and so load vis, flags and weights in parallel.
+    In older formats it causes an extra copy.
+
+    Parameters
+    ----------
+    dataset : :class:`katdal.DataSet`
+        Input dataset, possibly with an existing selection
+    indices : tuple
+        Index expression for subsetting the dataset
+    vis, weights, flags : array-like
+        Outputs, which must have the correct shape and type
+    """
+    if hasattr(dataset.vis, 'dataset'):
+        da.store([dataset.vis.dataset[indices],
```

Comment:
Does this do a `compute()` as well?

Reply:
No, because `compute()` generates new memory. This evaluates the graph but copies the chunks directly into the existing storage without assembling them into contiguous arrays.